app/src/main/res/layout/view_card_header.xml

```diff
@@ -35,7 +35,9 @@
         android:textSize="14sp"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintStart_toEndOf="@+id/view_card_header_image"
+        app:layout_constraintEnd_toStartOf="@+id/view_card_header_subtitle"
         app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintHorizontal_weight="1"
```

Comment:
Is this actually necessary, if we already specify `toStartOf` and `toEndOf`?

Reply:
It looks fine if we remove it, but I still think it would be better to set this because the `android:layout_width` is `0dp`.

Reply:
We have many other cases where the width or height is `0dp` because we have `toStartOf` and `toEndOf` constraints, all without needing `weight`.

Reply:
You convinced me. 👍